ui/packages/consul-ui/app/adapters/nspace.js

```diff
@@ -13,6 +13,10 @@ export default class NspaceAdapter extends Adapter {
         index,
       }}
     `;
+    if (once === 'once') {
+      await respond((headers, body) => delete headers['x-consul-index']);
+    }
+    return respond;
```

Comment:
If the HTTP headers don't have a `x-consul-index` on them then they don't block. That logic is done further into the codebase and has been around a long time and we use this approach elsewhere to disable blocking queries also. The `once === 'once'` is a little unfortunate here but it does the job with as little PR change as possible.